sonar-db/src/main/java/org/sonar/db/component/ComponentDto.java

```diff
@@ -75,6 +130,22 @@ public ComponentDto setUuid(String uuid) {
     return this;
   }
 
+  public String getUuidPath() {
```

Comment:
This method is used only in DAO and ComponentDto itself. We are so close to make it package protected. If only ComponentDto and DAO where in the same package...